app/models/post.rb

```diff
@@ -1,4 +1,8 @@
 class Post < ApplicationRecord
+  validates :title, presence: true, length: { maximum: 1000, too_long: '1000 characters is the maximum allowed' }
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, according to the requirements, the title's length must not exceed 250 characters and not 1000. Please consider changing this! 🤓 🔧 

![image](https://user-images.githubusercontent.com/77069011/222831415-11944e95-f180-459a-9d97-f604d78c4ebd.png)

```suggestion
  validates :title, presence: true, length: { maximum: 250, too_long: '250 characters is the maximum allowed' }
```